pay-api/src/pay_api/utils/handlers.py

```diff
@@ -0,0 +1,29 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Callbacks and signal trapping used in the main loop."""
+from flask import current_app
+
+
+async def error_cb(e):
+    """Emit error message to the log stream."""
+    current_app.logger.error(e)
+    raise e
+
+
+async def closed_cb():
+    """Exit the session after the NATS connection is closed."""
+    current_app.logger.info('Connection to NATS is closed.')
+    # my_loop = asyncio.get_running_loop()
```

Comment:
Could this be removed?

Reply:
I wanted some mechanism to know in logs when the connection closes. Technically it would happen after publishing to Queue, but since this is the first time working with nats, just want to see at-least in logs if there are any other chance of closing it. If you are okay, I would say we can remove the callback in next iteration, once we see this in action.